backend/Services/SightingService.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections.Generic;
+using System.Linq;
+using WhaleSpotting.Models.Database;
+using WhaleSpotting.Models.Request;
+using WhaleSpotting.Repositories;
+using WhaleSpotting.Models.Response;
+
+namespace WhaleSpotting.Services
+{
+    public interface ISightingService
+    {
+        List<SightingResponse> GetAllSightings();
+    }
+
+    public class SightingService : ISightingService
+    {
+        private ISightingRepo _sightingRepo;
+
+        public SightingService(ISightingRepo sightingRepo)
+        {
+            _sightingRepo = sightingRepo;
+        }
+
+        public List<SightingResponse> GetAllSightings()
+        {
+            var sightings = _sightingRepo.GetAllSightings();
+            return sightings.Select(sighting => new SightingResponse(sighting)).ToList();
```

Comment:
You shouldn't convert to a Response here, you should convert in the Controller - the Service should deal mostly in C# classes (this is slightly subjective, but will be consistent with the rest of the site this way!)